script.js

```diff
@@ -1,12 +1,208 @@
-//You can edit ALL of the code here
+const selectElement = document.getElementById("choose");
+const selectElementShow = document.getElementById("choose-show");
+const searchElement = document.getElementById("search-input");
+const numberOfEpisodes = document.getElementById("displaying-episodes");
+const rootElem = document.getElementById("root");
+const rootElemShow = document.getElementById("root-shows");
+
+const allEpisodes = getAllEpisodes();
+const shows = getAllShows();
+const oneShow = getOneShow();
+const starUrl =
+  "https://www.citypng.com/public/uploads/preview/hd-orange-star-transparent-background-116590439195j8drukpqu.png";
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
-}
+  searchShows();
+  addOptionsShow();
+  chooseShow(selectElementShow, "change");
+  numberOfEpisodes.innerText = `found ${shows.length} shows`;
+  showAllShows(shows);
 
+  // fetchEpisodes();
+}
+// level 100
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  selectElementShow.style.display = "inline";
+  selectElement.style.display = "inline";
+  rootElemShow.textContent = "";
+  episodeList.forEach((episode) => addEpisode(episode));
```

Comment:
Is there an array function you could use instead of `forEach` here?

Can you look for replacements for your other calls to `forEach` too?